logger/logger.go

```diff
@@ -0,0 +1,62 @@
+package logger
```

Comment:
Here is an example of a context-integrated logger that is threadsafe: https://github.com/docker/containerd/blob/master/log/context.go#L28.

However, please let me know what the intent of this package and we can get it into shape.